pallets/loans-ref/src/migrations.rs

```diff
@@ -0,0 +1,91 @@
+#[cfg(feature = "try-runtime")]
+use frame_support::ensure;
+use frame_support::{
+	pallet_prelude::ValueQuery,
+	storage, storage_alias,
+	traits::{Get, OnRuntimeUpgrade},
+	weights::Weight,
+	Blake2_128Concat,
+};
+#[cfg(feature = "try-runtime")]
+use sp_std::vec::Vec;
+
+use crate::*;
+
+mod v0 {
+	use super::*;
+
+	/// This storage comes from the previous pallet loans.
+	/// It is used as an indicator that the previous pallet loans still exists.
+	/// If this storage is not found, the nuking process is aborted.
+	#[storage_alias]
+	pub(crate) type NextLoanId<T: Config> =
+		StorageMap<Pallet<T>, Blake2_128Concat, PoolIdOf<T>, u128, ValueQuery>;
```

Comment:
Do we get the right prefix as `pallet-loans-ref` and `pallet-laons` are both using the same prefix (i.e. `Loans`)?